File: solutions/python/nth-prime/1/nth_prime.py
def prime(number):

    if number == 0:
        raise ValueError("there is no zeroth prime")

    
    result = []
    candidate = 2

    while len(result) < number:

        prime_number = True

        for i in range(2, candidate):

            if candidate % i == 0:
                prime_number = False
                break

        if prime_number:
            result.append(candidate)

        candidate += 1

    return result[-1]

File: solutions/javascript/nth-prime/1/nth-prime.js
//
// This is only a SKELETON file for the 'Nth Prime' exercise. It's been provided as a
// convenience to get you started writing code faster.
//

export const prime = (n) => {
  let count=0;
  let num=1;
  if (n<1)throw new Error('there is no zeroth prime');

  while (count<n){
    num++;
  let isPrime = true;

    for (let i=2;i*i<=num;i++){
      if(num % i ===0){
        isPrime=false;
        break;
      }
    }
    if (isPrime) {
        count++;
    }
}

return num;

  
};
